src/app/features/notification/notification.service.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { TestBed } from "@angular/core/testing";
+import { provideHttpClientTesting } from "@angular/common/http/testing";
+import { provideHttpClient } from "@angular/common/http";
+import { NotificationService } from "./notification.service";
+
+describe("NotificationService", () => {
```

Comment:
We should  implement unit tests for this service. If you are unsure about jasmine testing and/or mocking Http results, let me know to get this started together (or check whether Bappi or Vaibhav can provide some input).

Maybe we can start with collecting a checklist of test cases that can occur here:
- [ ] init with config not available
- [ ] init with config available -> should init firebase
- [ ] getFcmToken
   - [ ]  with / without / broken cookie
   - [ ] successful / failing api request
- ...